tiled/client/base.py

```diff
@@ -370,6 +380,99 @@ def update_metadata(self, metadata=None, specs=None):
 
         This is subject to change or removal without notice
 
+        Parameters
+        ----------
+        metadata : dict, optional
+            User metadata. May be nested. Must contain only basic types
+            (e.g. numbers, strings, lists, dicts) that are JSON-serializable.
+            Hint:
+                You can use `metadata_copy()` to retrieve a mutable copy of
+                the current metadata that can be passed here with modifications.
+            Note:
+                You only need to specify the relative paths in the nested
+                structure. `metadata_copy()` is only meant as an assistance.
+                `update_metadata()` implements a similar approach to json merge
+                patch (RFC 7386 https://datatracker.ietf.org/doc/html/rfc7386).
+                However, None (or null) does not take a special meaning,
+                although `tiled.client.metadata_update.DELETE_KEY` does!
+        specs : List[str], optional
+            List of names that are used to label that the data and/or metadata
+            conform to some named standard specification.
+
```

Comment:
Consider adding:

```
See Also
--------
patch_metadata
replace_metadata
```

And likewise on the other methods, to mutually cross-reference